extensions/apicurio-registry-avro/deployment/src/main/java/io/quarkus/apicurio/registry/avro/DevServicesApicurioRegistryProcessor.java

```diff
@@ -254,23 +253,17 @@ public int hashCode() {
     }
 
     private static final class ApicurioRegistryContainer extends GenericContainer<ApicurioRegistryContainer> {
-        private final int port;
+        private final int fixedExposedPort;
         private final boolean useSharedNetwork;
 
         private String hostName = null;
 
         private ApicurioRegistryContainer(DockerImageName dockerImageName, int fixedExposedPort, String serviceName,
                 boolean useSharedNetwork) {
             super(dockerImageName);
-            this.port = fixedExposedPort;
+            this.fixedExposedPort = fixedExposedPort;
             this.useSharedNetwork = useSharedNetwork;
-            withNetwork(Network.SHARED);
```

Comment:
Coming back to this after encountering a different problem: Why was this call removed? Do you remember?
It seems wrong to me...

Reply:
It was made consistent with the other DevServices and enabled [here](https://github.com/quarkusio/quarkus/pull/21863/files/ed96a827c1c71c75980f6984ace010c4404c2ee7#diff-5ed48b8e40721cbd73d201179ed62a7bedfacf8116874a91ed32b3581b0b4e52R281) if `useSharedNetwork` is enabled.

Reply:
Apicurio was always on the shared network; even if `useSharedNetwork` was false... perhaps it needed to be that way, perhaps it did not.. but there was nothing to suggest either way.. so yes use of `useSharedNetwork` was made "consistent" across all DevServices..... like I said yesterday, looking Quarkus is both fun and dangerous!

Reply:
But that part doesn't call `withNetwork` or `setNetworkAliases` unless I am mistaken. Am I missing something?

Reply:
Ah sorry, my bad - I see where the call was moved to

Reply:
Yes, it's a "nice" method call isn't it... took me a while before I stepped into it too!